package-lock.json

```diff
@@ -1,12669 +0,0 @@
-{
-  "name": "coderscamp2020.project.fullstack-node-react.organizationapp",
-  "version": "1.0.0",
-  "lockfileVersion": 1,
-  "requires": true,
-  "dependencies": {
-    "@babel/code-frame": {
```

Comment:
Why this file was deleted?
Probably this is cause of failing ci check

Reply:
I've got some weird problems with project building process - exactly with Jest (script found version 26.6.3 but react works with 26.6.0). 
I'll add it back soon ;)

Reply:
I also had problem with Jest and solved it setting env variable
`SKIP_PREFLIGHT_CHECK=true`

Reply:
right, it works, thanks